Example/SnapshotTests/BPKMapAnnotationSnapshotTest.m

```diff
@@ -147,7 +148,7 @@ -(UIView *)createLongTitleCalloutViewWithIcon:(BOOL)icon {
     annotationView.translatesAutoresizingMaskIntoConstraints = NO;
 
     [NSLayoutConstraint activateConstraints:@[
-        [annotationView.leadingAnchor constraintEqualToAnchor:outerView.leadingAnchor constant:65],
+        [annotationView.leadingAnchor constraintEqualToAnchor:outerView.leadingAnchor constant:55],
```

Comment:
Before this was different to the value below to make up for the inner view not being centred.